examples/localization/src/globals/Header/Component.client.tsx

```diff
@@ -45,7 +45,7 @@ export const HeaderClient: React.FC<HeaderClientProps> = ({ header }) => {
       className="container relative z-20 py-8 flex justify-end gap-2"
       {...(theme ? { 'data-theme': theme } : {})}
     >
-      <Link href="/" className="mr-auto">
+      <Link href="/" className="me-auto">
```

Comment:
Why this change?

Reply:
Explaining here for consistency. `me-auto` means `margin-inline-end: auto`, essentially the equivalent of `margin-right: auto` but respecting RTL direction